src/pages/workspaces/workspace/jobHistory/CallTable.js

```diff
@@ -0,0 +1,101 @@
+    h(AutoSizer, { disableHeight: true }, [
+      ({ width }) => h(FlexTable, {
+        height: tableHeight({ actualRows: callObjects.length, maxRows: 10.5 }), // The half-row here hints at there being extra rows if scrolled
+        width: _.max([width, 0]),
+        rowCount: callObjects.length,
+        noContentMessage: 'No matching calls',
+        columns: [
+          {
+            size: { basis: 100, grow: 0 },
+            headerRenderer: () => 'Index',
+            cellRenderer: ({ rowIndex }) => {
+              const { shardIndex } = callObjects[rowIndex]
+              return shardIndex >= 0 ? shardIndex : 'N/A'
+            }
+          }, {
+            size: { basis: 100, grow: 0 },
+            headerRenderer: () => 'Attempt',
+            cellRenderer: ({ rowIndex }) => {
+              const { attempt } = callObjects[rowIndex]
+              return attempt
+            }
+          }, {
+            size: { basis: 200, grow: 2 },
+            headerRenderer: () => 'Status',
+            cellRenderer: ({ rowIndex }) => {
+              const { executionStatus } = callObjects[rowIndex]
+              return h(TooltipCell, [makeCromwellStatusLine(executionStatus)])
+            }
+          }, {
+            size: { basis: 200, grow: 2 },
+            headerRenderer: () => 'Start',
+            cellRenderer: ({ rowIndex }) => {
+              const { start } = callObjects[rowIndex]
+              return h(TooltipCell, [start ? Utils.makeCompleteDate(start) : 'N/A'])
+            }
+          }, {
+            size: { basis: 200, grow: 2 },
+            headerRenderer: () => 'End',
+            cellRenderer: ({ rowIndex }) => {
+              const { end } = callObjects[rowIndex]
+              return h(TooltipCell, [end ? Utils.makeCompleteDate(end) : 'N/A'])
+            }
+          },
+          {
+            size: { basis: 200, grow: 2 },
+            headerRenderer: () => 'Call Caching Result',
+            cellRenderer: ({ rowIndex }) => {
+              const { callCaching: { effectiveCallCachingMode, result } = {} } = callObjects[rowIndex]
+              if (effectiveCallCachingMode === 'ReadAndWriteCache' || effectiveCallCachingMode === 'ReadCache') {
+                return result ? h(TooltipCell, [result]) : div({ style: { color: colors.dark(0.7) } }, ['No Information'])
+              } else if (effectiveCallCachingMode === 'WriteCache') {
+                return div({ style: { color: colors.dark(0.7) } }, ['Written but not read'])
+              } else {
+                return div({ style: { color: colors.dark(0.7) } }, [effectiveCallCachingMode])
+              }
+            }
+          },
+          {
+            size: { basis: 200, grow: 2 },
+            headerRenderer: () => 'Links',
+            cellRenderer: ({ rowIndex }) => {
+              const { failures, shardIndex: index, attempt } = callObjects[rowIndex]
+              const failureCount = _.size(failures)
+              return [
```

Comment:
This is returning an array, which is triggering a key warning. You can return `null` to render nothing, so we could just return a ternary expression with the `Link` or `null`